site/SiteFooter.tsx

```diff
@@ -262,7 +264,9 @@ export const SiteFooter = (props: SiteFooterProps) => (
             <script src={webpackUrl("owid.js", props.baseUrl)} />
             <script
                 dangerouslySetInnerHTML={{
-                    __html: `window.runSiteFooterScripts()`, // todo: gotta be a better way.
+                    __html: `window.runSiteFooterScripts(${JSON.stringify(
+                        props.context
+                    )})`, // todo: gotta be a better way.
```

Comment:
Does this count as `do`ing? 😄

Reply:
ahah I wouldn't say so, I think the author of the todo targeted the `dangerouslySetInnerHTML()` bit, but I don't know for sure.